experimental/trusted_shuffler/server/src/http.rs

```diff
@@ -76,8 +96,31 @@ async fn hyper_to_trusted_shuffler_request(hyper_request: Request<Body>) -> Trus
     let body = hyper::body::to_bytes(body)
         .await
         .expect("Couldn't read request body");
+    log::info!("Body {:?}", body);
+
+    let hyper_headers = parts.headers;
+
+    // Keep the minimal headers the backend requires to handle the request.
+    let mut minimal_headers = hyper::HeaderMap::new();
```

Comment:
I still think we should just copy all the headers and the trailers eventually.

Reply:
I think we want to have control on what we copy to show we don't have identifying information in there. So I'd argue not even for a `HeaderMap` in `TrustedShufflerRequest` but even more fine grained control. But let's keep thinking on this!